src/types/LinksPath.ts

```diff
@@ -0,0 +1,5 @@
+export enum LinksPath {
+  ALL = '/',
+  ACTIVE = '/active',
+  COMPLETED = '/completed',
```

Comment:
```suggestion
export enum LinksPath {
  All = '/',
  Active = '/active',
  Completed = '/completed',
```
Enum properties should be in PascalCase, not upper case